components/gallery/GalleryItemDescription.vue

```diff
@@ -85,14 +85,35 @@
       </div>
 
       <div
-        v-if="nft?.recipient"
-        class="is-flex is-justify-content-space-between">
-        <p>{{ $t('transfers.recipient') }}</p>
-        <nuxt-link
-          :to="`/${urlPrefix}/u/${nft?.recipient}`"
-          class="has-text-link">
-          <Identity ref="identity" :address="nft?.recipient" />
-        </nuxt-link>
+        v-if="recipient"
+        class="recipient is-flex is-justify-content-space-between">
+        <template v-if="Array.isArray(recipient) && recipient.length > 1">
+          <p>{{ $t('transfers.recipients') }}</p>
+          <ul>
```

Comment:
You could have use `<ol>` tag here to display numbers

Reply:
there was an issue with displaying numbers and flex li items i kept it

Reply:
please can you try to fix it?

Reply:
you mean by using `<ol>` instead of `<ul>` with the index?